bot/cogs/off_topic_names.py

```diff
@@ -49,9 +49,13 @@ class OffTopicName(Converter):
         seconds_to_sleep = (next_midnight - datetime.utcnow()).seconds + 1
         await asyncio.sleep(seconds_to_sleep)
 
-        channel_0_name, channel_1_name, channel_2_name = await bot.api_client.get(
-            'bot/off-topic-channel-names', params={'random_items': 3}
-        )
+        try:
+            channel_0_name, channel_1_name, channel_2_name = await bot.api_client.get(
+                'bot/off-topic-channel-names', params={'random_items': 3}
+            )
+        except bot.api.ResponseCodeError as e:
```

Comment:
This won't work, since you're now trying to access the `api` attribute of the `Bot` object with the name `bot`, not the `bot.api` module:

```py
During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "<string>", line 9, in func
  File "/home/sebastiaan/pydis/repositories/djangobot/bot/cogs/off_topic_names.py", line 56, in update_names
    except bot.api.ResponseCodeError as e:
AttributeError: 'Bot' object has no attribute 'api'
```

One way to fix this is to import `ResponseCodeError` from `bot.api`:
```py
from bot.api import ResponseCodeError
```
and then using `ResponseCodeError` directly:
```suggestion
        except ResponseCodeError as e:
```

The exception will then be handled correctly:
```
Sep 24 11:27:52 pd.beardfist.com Bot: |          bot.cogs.off_topic_names |    ERROR | Failed to get new off topic channel names: code 404
``` 

### Note: For the suggested change to work, you do need to include that import at the top of the file as well.

Reply:
Sorry I didn't accepted your changes but the fix has been done because of github mobile stupid UI :)